man/readWQPsummary.Rd

```diff
@@ -68,9 +68,10 @@ dane_county_data <- readWQPsummary(countycode = "US:55:025",
 
```

Comment:
Can you add a descriptor for these examples? e.g 'Obtain a summary table of all stream water quality data for the state of New Jersey from the past 5 years'